### PR TITLE
To do dynamic

### PR DESCRIPTION
**Pull request by NelsonAraujo93**
In this pull request i followed the next steps:

- Create sec Folder and Index.js, style.css & index.html files.
- Add style.css to wepback.conf.js.
- Create class todoList in index.js and methods to add, update, and print dynamicly.
- Create a function to add on Enter key press.
- Create README file.
- Run Linters.
- Fix linters.

### CODE REVIEW

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @NelsonAraujo93 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design
✔️  Good PR title and Description



## Required Changes ♻️
- Your package.json should have  scripts like   `start` to enable you start the project with `npm start`  and enable live reloading,  please see [webpack documentation](https://webpack.js.org/guides/development/#using-webpack-dev-server) to see how to set it up  👍🏿 


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json [hunk 1]
```diff
@@ -19,6 +19,19 @@
...
+    "sass": "^1.57.1",
+    "sass-loader": "^13.2.0",
+    "style-loader": "^3.3.1",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0",
     "webpack": "^5.75.0",
     "webpack-cli": "^5.0.1"
   },
```
Comment (Owusu-Desmond) [inline package.json:37]: - [ ] As requested by the previous reviewer, your package.json should have scripts like `start` to enable you to start the project with npm start and enable live reloading, please see [webpack documentation](https://webpack.js.org/guides/development/#using-webpack-dev-server) to see how to set it up 👍🏿. For now, I can't start your project running `npm start` as you can see below :point_down: 
Screenshot from 2023-01-18 19-16-22

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @NelsonAraujo93 

You've done well implementing some of the requested changes, but there are still some changes you still need to work on before you move on to the next project but you are almost there.



 ## Highlights :+1: 
:heavy_check_mark: Good commit history.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Your project is well documented

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @NelsonAraujo93, 

To highlight 👍🏽 
✔️ Your project runs smoothly

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**